src/components/ThumbnailImage.tsx

```diff
@@ -1,51 +1,37 @@
 import lodashClamp from 'lodash/clamp';
-import PropTypes from 'prop-types';
 import React, {useCallback, useState} from 'react';
-import {Dimensions, View} from 'react-native';
+import {Dimensions, StyleProp, View, ViewStyle} from 'react-native';
 import useWindowDimensions from '@hooks/useWindowDimensions';
 import * as DeviceCapabilities from '@libs/DeviceCapabilities';
 import styles from '@styles/styles';
 import * as StyleUtils from '@styles/StyleUtils';
 import ImageWithSizeCalculation from './ImageWithSizeCalculation';
 
-const propTypes = {
+type ThumbnailImageProps = {
     /** Source URL for the preview image */
-    previewSourceURL: PropTypes.string.isRequired,
+    previewSourceURL: string;
 
     /** Any additional styles to apply */
-    // eslint-disable-next-line react/forbid-prop-types
-    style: PropTypes.any,
+    style?: StyleProp<ViewStyle>;
 
     /** Whether the image requires an authToken */
-    isAuthTokenRequired: PropTypes.bool.isRequired,
+    isAuthTokenRequired: boolean;
 
     /** Width of the thumbnail image */
-    imageWidth: PropTypes.number,
+    imageWidth?: number;
 
     /** Height of the thumbnail image */
-    imageHeight: PropTypes.number,
+    imageHeight?: number;
 
     /** Should the image be resized on load or just fit container */
-    shouldDynamicallyResize: PropTypes.bool,
-};
-
-const defaultProps = {
-    style: {},
-    imageWidth: 200,
-    imageHeight: 200,
-    shouldDynamicallyResize: true,
+    shouldDynamicallyResize?: boolean;
 };
 
 /**
  * Compute the thumbnails width and height given original image dimensions.
- *
- * @param {Number} width - Width of the original image.
- * @param {Number} height - Height of the original image.
- * @param {Number} windowHeight - Height of the device/browser window.
- * @returns {Object} - Object containing thumbnails width and height.
```

Comment:
The comments have descriptions, let's bring them back

Reply:
reverted